config.example.py

```diff
@@ -31,7 +33,9 @@
       'db_review': 'db/*',
       'css_review': 'app/assets/stylesheets/global/*',
       'js_review': 'node_modules/*',
-      'ops_review': ['config/nginx/*', 'config/settings/*']
+      'ops_review': ['config/nginx/*', 'config/settings/*'],
+      # Only matches files in this folder, not any subfolder
+      'test_review': re.compile(r'test\/[a-z_]+.rb'),
```

Comment:
You're just trying to match files who are children of the app's `test` directory? Like `test_helper.rb`, `selenium_helper.rb`, and `selenium_driver.rb`?

Right now, it seems the pattern will also match files like `vendor/cache/prototype-rails-0fed929ff48c/test/assert_select_test.rb`, since you're not specifying that the pattern start matching only at the beginning of the path.